XROLE/storeManus
# feat(consume endpoint): consume get Attendant sales endpoint

#### What does this pull request do?
- Implement fetching an Attendant sales recored.
#### Description of task to be completed?
- [x] consume get Attendant sales API
#### How should this be manually tested?
- clone the repo, run npm install and then npm start
- sign in as an attendant
- create sales record if you have not created any before
- click of 'sales record' to see your sales record.
#### What are the relivant pivotal tracker stories?
[Finishes #162195542]
